### ISSUE TITLE
Unable to install under Centos 8

### ISSUE THREAD

**Issue by zdm** [vlet/p5-Protocol-HTTP2#10]
Centos 8 - tests are failed
Centos 7, Fedora 30, 31 - OK

```
[root@d3a191917302 /]# cpanm Protocol::HTTP2 -v
cpanm (App::cpanminus) 1.7044 on perl 5.026003 built for x86_64-linux-thread-multi
Work directory is /root/.cpanm/work/1570966752.2652
You have make /usr/bin/make
You have /usr/bin/curl
You have /usr/bin/tar: tar (GNU tar) 1.30
Copyright (C) 2017 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by John Gilmore and Jay Fenlason.
You have /usr/bin/unzip
Searching Protocol::HTTP2 () on cpanmetadb ...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   989  100   989    0     0   8106      0 --:--:-- --:--:-- --:--:--  8173
--> Working on Protocol::HTTP2
Fetching http://www.cpan.org/authors/id/C/CR/CRUX/Protocol-HTTP2-1.09.tar.gz ... ##O=#   #                                                       ########################################################################################################################################## 100.0%
OK
Unpacking Protocol-HTTP2-1.09.tar.gz
Protocol-HTTP2-1.09/Build.PL
Protocol-HTTP2-1.09/Changes
Protocol-HTTP2-1.09/LICENSE
Protocol-HTTP2-1.09/META.json
Protocol-HTTP2-1.09/README.md
Protocol-HTTP2-1.09/cpanfile
Protocol-HTTP2-1.09/examples/client-anyevent.pl
Protocol-HTTP2-1.09/examples/client-io-socket-ssl.pl
Protocol-HTTP2-1.09/examples/client-tls-anyevent.pl
Protocol-HTTP2-1.09/examples/extract_huff_codes.pl
Protocol-HTTP2-1.09/examples/extract_static_table.pl
Protocol-HTTP2-1.09/examples/server-anyevent.pl
Protocol-HTTP2-1.09/examples/server-io-socket-ssl.pl
Protocol-HTTP2-1.09/examples/server-tls-anyevent.pl
Protocol-HTTP2-1.09/examples/test.crt
Protocol-HTTP2-1.09/examples/test.key
Protocol-HTTP2-1.09/lib/Protocol/HTTP2.pm
Protocol-HTTP2-1.09/lib/Protocol/HTTP2/Client.pm
Protocol-HTTP2-1.09/lib/Protocol/HTTP2/Connection.pm
Protocol-HTTP2-1.09/lib/Protocol/HTTP2/Constants.pm
Protocol-HTTP2-1.09/lib/Protocol/HTTP2/Frame.pm
Protocol-HTTP2-1.09/lib/Protocol/HTTP2/Frame/Continuation.pm
Protocol-HTTP2-1.09/lib/Protocol/HTTP2/Frame/Data.pm
Protocol-HTTP2-1.09/lib/Protocol/HTTP2/Frame/Goaway.pm
Protocol-HTTP2-1.09/lib/Protocol/HTTP2/Frame/Headers.pm
Protocol-HTTP2-1.09/lib/Protocol/HTTP2/Frame/Ping.pm
Protocol-HTTP2-1.09/lib/Protocol/HTTP2/Frame/Priority.pm
Protocol-HTTP2-1.09/lib/Protocol/HTTP2/Frame/Push_promise.pm
Protocol-HTTP2-1.09/lib/Protocol/HTTP2/Frame/Rst_stream.pm
Protocol-HTTP2-1.09/lib/Protocol/HTTP2/Frame/Settings.pm
Protocol-HTTP2-1.09/lib/Protocol/HTTP2/Frame/Window_update.pm
Protocol-HTTP2-1.09/lib/Protocol/HTTP2/HeaderCompression.pm
Protocol-HTTP2-1.09/lib/Protocol/HTTP2/Huffman.pm
Protocol-HTTP2-1.09/lib/Protocol/HTTP2/HuffmanCodes.pm
Protocol-HTTP2-1.09/lib/Protocol/HTTP2/Server.pm
Protocol-HTTP2-1.09/lib/Protocol/HTTP2/StaticTable.pm
Protocol-HTTP2-1.09/lib/Protocol/HTTP2/Stream.pm
Protocol-HTTP2-1.09/lib/Protocol/HTTP2/Trace.pm
Protocol-HTTP2-1.09/lib/Protocol/HTTP2/Upgrade.pm
Protocol-HTTP2-1.09/minil.toml
Protocol-HTTP2-1.09/t/00_compile.t
Protocol-HTTP2-1.09/t/01_HeaderCompression.t
Protocol-HTTP2-1.09/t/02_Huffman.t
Protocol-HTTP2-1.09/t/03_connection.t
Protocol-HTTP2-1.09/t/04_continuation.t
Protocol-HTTP2-1.09/t/05_trace.t
Protocol-HTTP2-1.09/t/06_upgrade.t
Protocol-HTTP2-1.09/t/07_ping.t
Protocol-HTTP2-1.09/t/08_priority.t
Protocol-HTTP2-1.09/t/09_client_server_tcp.t
Protocol-HTTP2-1.09/t/10_settings.t
Protocol-HTTP2-1.09/t/11_server_stream.t
Protocol-HTTP2-1.09/t/12_leaks.t
Protocol-HTTP2-1.09/t/13_request_with_body.t
Protocol-HTTP2-1.09/t/14_keepalive.t
Protocol-HTTP2-1.09/t/continuation.request.data
Protocol-HTTP2-1.09/t/lib/PH2ClientServerTest.pm
Protocol-HTTP2-1.09/t/lib/PH2Test.pm
Protocol-HTTP2-1.09/META.yml
Protocol-HTTP2-1.09/MANIFEST
Entering Protocol-HTTP2-1.09
Checking configure dependencies from META.json
Checking if you have Module::Build::Tiny 0.035 ... Yes (0.039)
Running Build.PL
Configuring Protocol-HTTP2-1.09 ... Creating new 'Build' script for 'Protocol-HTTP2' version '1.09'
OK
Checking dependencies from MYMETA.json ...
Checking if you have Test::LeakTrace 0 ... Yes (0.16)
Checking if you have Test::More 0.98 ... Yes (1.302135)
Checking if you have Net::SSLeay > 1.45 ... Yes (1.88)
Checking if you have Test::TCP 0 ... Yes (2.22)
Checking if you have MIME::Base64 3.11 ... Yes (3.15)
Checking if you have Scalar::Util 0 ... Yes (1.49)
Checking if you have AnyEvent 0 ... Yes (7.17)
Building and testing Protocol-HTTP2-1.09 ... cp lib/Protocol/HTTP2/HeaderCompression.pm blib/lib/Protocol/HTTP2/HeaderCompression.pm
cp lib/Protocol/HTTP2/Trace.pm blib/lib/Protocol/HTTP2/Trace.pm
cp lib/Protocol/HTTP2/Constants.pm blib/lib/Protocol/HTTP2/Constants.pm
cp lib/Protocol/HTTP2/HuffmanCodes.pm blib/lib/Protocol/HTTP2/HuffmanCodes.pm
cp lib/Protocol/HTTP2/Frame/Push_promise.pm blib/lib/Protocol/HTTP2/Frame/Push_promise.pm
cp lib/Protocol/HTTP2/Frame/Data.pm blib/lib/Protocol/HTTP2/Frame/Data.pm
cp lib/Protocol/HTTP2/Frame/Continuation.pm blib/lib/Protocol/HTTP2/Frame/Continuation.pm
cp lib/Protocol/HTTP2/StaticTable.pm blib/lib/Protocol/HTTP2/StaticTable.pm
cp lib/Protocol/HTTP2/Frame/Window_update.pm blib/lib/Protocol/HTTP2/Frame/Window_update.pm
cp lib/Protocol/HTTP2/Frame/Priority.pm blib/lib/Protocol/HTTP2/Frame/Priority.pm
cp lib/Protocol/HTTP2/Upgrade.pm blib/lib/Protocol/HTTP2/Upgrade.pm
cp lib/Protocol/HTTP2/Frame/Goaway.pm blib/lib/Protocol/HTTP2/Frame/Goaway.pm
cp lib/Protocol/HTTP2/Server.pm blib/lib/Protocol/HTTP2/Server.pm
cp lib/Protocol/HTTP2/Frame/Headers.pm blib/lib/Protocol/HTTP2/Frame/Headers.pm
cp lib/Protocol/HTTP2/Connection.pm blib/lib/Protocol/HTTP2/Connection.pm
cp lib/Protocol/HTTP2/Client.pm blib/lib/Protocol/HTTP2/Client.pm
cp lib/Protocol/HTTP2/Frame/Ping.pm blib/lib/Protocol/HTTP2/Frame/Ping.pm
cp lib/Protocol/HTTP2.pm blib/lib/Protocol/HTTP2.pm
cp lib/Protocol/HTTP2/Huffman.pm blib/lib/Protocol/HTTP2/Huffman.pm
cp lib/Protocol/HTTP2/Frame/Rst_stream.pm blib/lib/Protocol/HTTP2/Frame/Rst_stream.pm
cp lib/Protocol/HTTP2/Frame.pm blib/lib/Protocol/HTTP2/Frame.pm
cp lib/Protocol/HTTP2/Frame/Settings.pm blib/lib/Protocol/HTTP2/Frame/Settings.pm
cp lib/Protocol/HTTP2/Stream.pm blib/lib/Protocol/HTTP2/Stream.pm
t/00_compile.t ............ ok
t/01_HeaderCompression.t .. ok
t/02_Huffman.t ............ ok
t/03_connection.t ......... ok
t/04_continuation.t ....... ok
t/05_trace.t .............. ok
t/06_upgrade.t ............ ok
t/07_ping.t ............... ok
t/08_priority.t ........... ok
t/09_client_server_tcp.t ..
    #   Failed test 'no errors'
    #   at t/09_client_server_tcp.t line 115.
    #          got: 'Some problem with SSL CTX: examples/test.crt: failed to use local certificate chain (cert_file or cert) at t/lib/PH2ClientServerTest.pm line 45.
    #
    #  at /usr/local/lib64/perl5/AnyEvent/Socket.pm line 1266.
    # '
    #     expected: ''

    #   Failed test 'no errors'
    #   at t/09_client_server_tcp.t line 115.

    #   Failed test 'no errors'
    #   at t/09_client_server_tcp.t line 115.
    #          got: 'error occured
    #  at t/09_client_server_tcp.t line 77.
    # '
    #     expected: ''
    #          got: 'Some problem with SSL CTX: examples/test.crt: failed to use local certificate chain (cert_file or cert) at t/lib/PH2ClientServerTest.pm line 45.
    #
    #  at /usr/local/lib64/perl5/AnyEvent/Socket.pm line 1266.
    # '
    #     expected: ''
    # Looks like you planned 0 tests but ran 4.

    #   Failed test 'no errors'
    #   at t/09_client_server_tcp.t line 115.
    #          got: 'Some problem with SSL CTX: examples/test.crt: failed to use local certificate chain (cert_file or cert) at t/lib/PH2ClientServerTest.pm line 45.
    #
    #  at /usr/local/lib64/perl5/AnyEvent/Socket.pm line 1266.
    # '
    #     expected: ''
    # Looks like you planned 4 tests but ran 7.
    # Looks like you failed 3 tests of 7 run.

    #   Failed test 'no errors'
    #   at t/09_client_server_tcp.t line 115.
    #          got: 'error occured
    #  at t/09_client_server_tcp.t line 77.
    # '
    #     expected: ''
    # Looks like you planned 4 tests but ran 9.
    # Looks like you failed 5 tests of 9 run.
t/09_client_server_tcp.t .. 1/?
#   Failed test 'client/server'
#   at t/09_client_server_tcp.t line 117.

#   Failed test 'client/server'
#   at t/09_client_server_tcp.t line 117.

#   Failed test 'client/server'
#   at t/09_client_server_tcp.t line 117.
# Looks like you failed 3 tests of 3.
t/09_client_server_tcp.t .. skipped: (no reason given)
t/10_settings.t ........... ok
t/11_server_stream.t ...... ok
t/12_leaks.t .............. ok
t/13_request_with_body.t .. ok
t/14_keepalive.t .......... ok

Test Summary Report
-------------------
t/09_client_server_tcp.t (Wstat: 768 Tests: 3 Failed: 3)
  Failed tests:  1-3
  Non-zero exit status: 3
  Parse errors: Plan (1..0) must be at the beginning or end of the TAP output
                More than one plan found in TAP output
                More than one plan found in TAP output
                Bad plan.  You planned 0 tests but ran 3.
Files=15, Tests=37,  1 wallclock secs ( 0.03 usr  0.05 sys +  0.77 cusr  0.29 csys =  1.14 CPU)
Result: FAIL
FAIL
```

**Comment by vlet** [comment]
Thank you for report. Can you send the same output, but using sources from issue10 branch of this repo?
cpanm https://github.com/vlet/p5-Protocol-HTTP2/archive/issue10.tar.gz -v

**Comment by zdm** [comment]
Same errors.
```
[root@99cd49678131 /]# cpanm https://github.com/vlet/p5-Protocol-HTTP2/archive/issue10.tar.gz -v
cpanm (App::cpanminus) 1.7044 on perl 5.026003 built for x86_64-linux-thread-multi
Work directory is /root/.cpanm/work/1571045759.1512
You have make /usr/bin/make
You have /usr/bin/curl
You have /usr/bin/tar: tar (GNU tar) 1.30
Copyright (C) 2017 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by John Gilmore and Jay Fenlason.
You have /usr/bin/unzip
--> Working on https://github.com/vlet/p5-Protocol-HTTP2/archive/issue10.tar.gz
Fetching https://github.com/vlet/p5-Protocol-HTTP2/archive/issue10.tar.gz ... ##O=#   #                                                           -=O=-    #     #       #        #
OK
Unpacking issue10.tar.gz
p5-Protocol-HTTP2-issue10/
p5-Protocol-HTTP2-issue10/.gitignore
p5-Protocol-HTTP2-issue10/.travis.yml
p5-Protocol-HTTP2-issue10/Build.PL
p5-Protocol-HTTP2-issue10/Changes
p5-Protocol-HTTP2-issue10/LICENSE
p5-Protocol-HTTP2-issue10/META.json
p5-Protocol-HTTP2-issue10/README.md
p5-Protocol-HTTP2-issue10/cpanfile
p5-Protocol-HTTP2-issue10/examples/
p5-Protocol-HTTP2-issue10/examples/client-anyevent.pl
p5-Protocol-HTTP2-issue10/examples/client-io-socket-ssl.pl
p5-Protocol-HTTP2-issue10/examples/client-tls-anyevent.pl
p5-Protocol-HTTP2-issue10/examples/extract_huff_codes.pl
p5-Protocol-HTTP2-issue10/examples/extract_static_table.pl
p5-Protocol-HTTP2-issue10/examples/server-anyevent.pl
p5-Protocol-HTTP2-issue10/examples/server-io-socket-ssl.pl
p5-Protocol-HTTP2-issue10/examples/server-tls-anyevent.pl
p5-Protocol-HTTP2-issue10/examples/test.crt
p5-Protocol-HTTP2-issue10/examples/test.key
p5-Protocol-HTTP2-issue10/lib/
p5-Protocol-HTTP2-issue10/lib/Protocol/
p5-Protocol-HTTP2-issue10/lib/Protocol/HTTP2.pm
p5-Protocol-HTTP2-issue10/lib/Protocol/HTTP2/
p5-Protocol-HTTP2-issue10/lib/Protocol/HTTP2/Client.pm
p5-Protocol-HTTP2-issue10/lib/Protocol/HTTP2/Connection.pm
p5-Protocol-HTTP2-issue10/lib/Protocol/HTTP2/Constants.pm
p5-Protocol-HTTP2-issue10/lib/Protocol/HTTP2/Frame.pm
p5-Protocol-HTTP2-issue10/lib/Protocol/HTTP2/Frame/
p5-Protocol-HTTP2-issue10/lib/Protocol/HTTP2/Frame/Continuation.pm
p5-Protocol-HTTP2-issue10/lib/Protocol/HTTP2/Frame/Data.pm
p5-Protocol-HTTP2-issue10/lib/Protocol/HTTP2/Frame/Goaway.pm
p5-Protocol-HTTP2-issue10/lib/Protocol/HTTP2/Frame/Headers.pm
p5-Protocol-HTTP2-issue10/lib/Protocol/HTTP2/Frame/Ping.pm
p5-Protocol-HTTP2-issue10/lib/Protocol/HTTP2/Frame/Priority.pm
p5-Protocol-HTTP2-issue10/lib/Protocol/HTTP2/Frame/Push_promise.pm
p5-Protocol-HTTP2-issue10/lib/Protocol/HTTP2/Frame/Rst_stream.pm
p5-Protocol-HTTP2-issue10/lib/Protocol/HTTP2/Frame/Settings.pm
p5-Protocol-HTTP2-issue10/lib/Protocol/HTTP2/Frame/Window_update.pm
p5-Protocol-HTTP2-issue10/lib/Protocol/HTTP2/HeaderCompression.pm
p5-Protocol-HTTP2-issue10/lib/Protocol/HTTP2/Huffman.pm
p5-Protocol-HTTP2-issue10/lib/Protocol/HTTP2/HuffmanCodes.pm
p5-Protocol-HTTP2-issue10/lib/Protocol/HTTP2/Server.pm
p5-Protocol-HTTP2-issue10/lib/Protocol/HTTP2/StaticTable.pm
p5-Protocol-HTTP2-issue10/lib/Protocol/HTTP2/Stream.pm
p5-Protocol-HTTP2-issue10/lib/Protocol/HTTP2/Trace.pm
p5-Protocol-HTTP2-issue10/lib/Protocol/HTTP2/Upgrade.pm
p5-Protocol-HTTP2-issue10/minil.toml
p5-Protocol-HTTP2-issue10/t/
p5-Protocol-HTTP2-issue10/t/00_compile.t
p5-Protocol-HTTP2-issue10/t/01_HeaderCompression.t
p5-Protocol-HTTP2-issue10/t/02_Huffman.t
p5-Protocol-HTTP2-issue10/t/03_connection.t
p5-Protocol-HTTP2-issue10/t/04_continuation.t
p5-Protocol-HTTP2-issue10/t/05_trace.t
p5-Protocol-HTTP2-issue10/t/06_upgrade.t
p5-Protocol-HTTP2-issue10/t/07_ping.t
p5-Protocol-HTTP2-issue10/t/08_priority.t
p5-Protocol-HTTP2-issue10/t/09_client_server_tcp.t
p5-Protocol-HTTP2-issue10/t/10_settings.t
p5-Protocol-HTTP2-issue10/t/11_server_stream.t
p5-Protocol-HTTP2-issue10/t/12_leaks.t
p5-Protocol-HTTP2-issue10/t/13_request_with_body.t
p5-Protocol-HTTP2-issue10/t/14_keepalive.t
p5-Protocol-HTTP2-issue10/t/continuation.request.data
p5-Protocol-HTTP2-issue10/t/lib/
p5-Protocol-HTTP2-issue10/t/lib/PH2ClientServerTest.pm
p5-Protocol-HTTP2-issue10/t/lib/PH2Test.pm
Entering p5-Protocol-HTTP2-issue10
Checking configure dependencies from META.json
Checking if you have Module::Build::Tiny 0.035 ... Yes (0.039)
Running Build.PL
Configuring Protocol-HTTP2-1.09 ... Creating new 'Build' script for 'Protocol-HTTP2' version '1.09'
OK
Checking dependencies from MYMETA.json ...
Checking if you have Scalar::Util 0 ... Yes (1.49)
Checking if you have Test::TCP 0 ... Yes (2.22)
Checking if you have AnyEvent 0 ... Yes (7.17)
Checking if you have MIME::Base64 3.11 ... Yes (3.15)
Checking if you have Test::LeakTrace 0 ... Yes (0.16)
Checking if you have Test::More 0.98 ... Yes (1.302135)
Checking if you have Net::SSLeay > 1.45 ... Yes (1.85)
Building and testing Protocol-HTTP2-1.09 ... cp lib/Protocol/HTTP2/Frame.pm blib/lib/Protocol/HTTP2/Frame.pm
cp lib/Protocol/HTTP2/Frame/Rst_stream.pm blib/lib/Protocol/HTTP2/Frame/Rst_stream.pm
cp lib/Protocol/HTTP2/Server.pm blib/lib/Protocol/HTTP2/Server.pm
cp lib/Protocol/HTTP2/Frame/Ping.pm blib/lib/Protocol/HTTP2/Frame/Ping.pm
cp lib/Protocol/HTTP2/Huffman.pm blib/lib/Protocol/HTTP2/Huffman.pm
cp lib/Protocol/HTTP2/Frame/Goaway.pm blib/lib/Protocol/HTTP2/Frame/Goaway.pm
cp lib/Protocol/HTTP2/Constants.pm blib/lib/Protocol/HTTP2/Constants.pm
cp lib/Protocol/HTTP2/Frame/Push_promise.pm blib/lib/Protocol/HTTP2/Frame/Push_promise.pm
cp lib/Protocol/HTTP2/StaticTable.pm blib/lib/Protocol/HTTP2/StaticTable.pm
cp lib/Protocol/HTTP2/Upgrade.pm blib/lib/Protocol/HTTP2/Upgrade.pm
cp lib/Protocol/HTTP2/Trace.pm blib/lib/Protocol/HTTP2/Trace.pm
cp lib/Protocol/HTTP2.pm blib/lib/Protocol/HTTP2.pm
cp lib/Protocol/HTTP2/Frame/Continuation.pm blib/lib/Protocol/HTTP2/Frame/Continuation.pm
cp lib/Protocol/HTTP2/Frame/Priority.pm blib/lib/Protocol/HTTP2/Frame/Priority.pm
cp lib/Protocol/HTTP2/Frame/Headers.pm blib/lib/Protocol/HTTP2/Frame/Headers.pm
cp lib/Protocol/HTTP2/HuffmanCodes.pm blib/lib/Protocol/HTTP2/HuffmanCodes.pm
cp lib/Protocol/HTTP2/Connection.pm blib/lib/Protocol/HTTP2/Connection.pm
cp lib/Protocol/HTTP2/Client.pm blib/lib/Protocol/HTTP2/Client.pm
cp lib/Protocol/HTTP2/Frame/Window_update.pm blib/lib/Protocol/HTTP2/Frame/Window_update.pm
cp lib/Protocol/HTTP2/Frame/Settings.pm blib/lib/Protocol/HTTP2/Frame/Settings.pm
cp lib/Protocol/HTTP2/Frame/Data.pm blib/lib/Protocol/HTTP2/Frame/Data.pm
cp lib/Protocol/HTTP2/Stream.pm blib/lib/Protocol/HTTP2/Stream.pm
cp lib/Protocol/HTTP2/HeaderCompression.pm blib/lib/Protocol/HTTP2/HeaderCompression.pm
t/00_compile.t ............ ok
t/01_HeaderCompression.t .. ok
t/02_Huffman.t ............ ok
t/03_connection.t ......... ok
t/04_continuation.t ....... ok
t/05_trace.t .............. ok
t/06_upgrade.t ............ ok
t/07_ping.t ............... ok
t/08_priority.t ........... ok
t/09_client_server_tcp.t ..
    #   Failed test 'no errors'
    #   at t/09_client_server_tcp.t line 115.
    #          got: 'Some problem with SSL CTX: examples/test.crt: failed to use local certificate chain (cert_file or cert) at t/lib/PH2ClientServerTest.pm line 45.
    #  1538: 1 - error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small
    #  at /usr/local/lib64/perl5/AnyEvent/Socket.pm line 1266.
    # '
    #     expected: ''

    #   Failed test 'no errors'
    #   at t/09_client_server_tcp.t line 115.

    #   Failed test 'no errors'
    #   at t/09_client_server_tcp.t line 115.
    #          got: 'error occured
    #  at t/09_client_server_tcp.t line 77.
    # '
    #     expected: ''
    #          got: 'Some problem with SSL CTX: examples/test.crt: failed to use local certificate chain (cert_file or cert) at t/lib/PH2ClientServerTest.pm line 45.
    #  1539: 1 - error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small
    #  at /usr/local/lib64/perl5/AnyEvent/Socket.pm line 1266.
    # '
    #     expected: ''
    # Looks like you planned 0 tests but ran 4.

    #   Failed test 'no errors'
    #   at t/09_client_server_tcp.t line 115.
    #          got: 'Some problem with SSL CTX: examples/test.crt: failed to use local certificate chain (cert_file or cert) at t/lib/PH2ClientServerTest.pm line 45.
    #  1540: 1 - error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small
    #  at /usr/local/lib64/perl5/AnyEvent/Socket.pm line 1266.
    # '
    #     expected: ''
    # Looks like you planned 4 tests but ran 7.
    # Looks like you failed 3 tests of 7 run.

    #   Failed test 'no errors'
    #   at t/09_client_server_tcp.t line 115.
    #          got: 'error occured
    #  at t/09_client_server_tcp.t line 77.
    # '
    #     expected: ''
    # Looks like you planned 4 tests but ran 9.
    # Looks like you failed 5 tests of 9 run.
t/09_client_server_tcp.t .. 1/?
#   Failed test 'client/server'
#   at t/09_client_server_tcp.t line 117.

#   Failed test 'client/server'
#   at t/09_client_server_tcp.t line 117.

#   Failed test 'client/server'
#   at t/09_client_server_tcp.t line 117.
# Looks like you failed 3 tests of 3.
t/09_client_server_tcp.t .. skipped: (no reason given)
t/10_settings.t ........... ok
t/11_server_stream.t ...... ok
t/12_leaks.t .............. ok
t/13_request_with_body.t .. ok
t/14_keepalive.t .......... ok

Test Summary Report
-------------------
t/09_client_server_tcp.t (Wstat: 768 Tests: 3 Failed: 3)
  Failed tests:  1-3
  Non-zero exit status: 3
  Parse errors: Plan (1..0) must be at the beginning or end of the TAP output
                More than one plan found in TAP output
                More than one plan found in TAP output
                Bad plan.  You planned 0 tests but ran 3.
Files=15, Tests=37,  2 wallclock secs ( 0.06 usr  0.05 sys +  1.40 cusr  0.45 csys =  1.96 CPU)
Result: FAIL
FAIL
! Installing https://github.com/vlet/p5-Protocol-HTTP2/archive/issue10.tar.gz failed. See /root/.cpanm/work/1571045759.1512/build.log for details. Retry with --force to force install it.
[root@99cd49678131 /]#

```

**Comment by vlet** [comment]
Ok. I've updated test.crt/test.key. Can you try again the same command?
cpanm https://github.com/vlet/p5-Protocol-HTTP2/archive/issue10.tar.gz -v

**Comment by zdm** [comment]
Now tests are passed without errors.

**Comment by vlet** [comment]
Thank you.

**Comment by zdm** [comment]
Could you, please, release the latest version on CPAN?

**Comment by vlet** [comment]
Released 1.10
